src/Network/HTTP/Download/Verified.hs

```diff
@@ -243,9 +243,10 @@ verifiedDownload
          :: HasTerm env
          => DownloadRequest
          -> Path Abs File -- ^ destination
+         -> Bool -- ^ whether to redownload or not if file exists
```

Comment:
Can this be added to `DownloadRequest` to avoid breaking the API here?

Reply:
Actually looks like it might still be an API breakage, but still nicer than a plain `Bool`.

Reply:
Yup sure